src/app/modules/submission/email-submission/ParsedResponsesObject.class.ts

```diff
@@ -152,7 +152,8 @@ export default class ParsedResponsesObject {
           // encrypt mode submissions with responses on unhidden fields
           // TODO(#780): Remove this once submission service is separated into
           // Email and Encrypted services
-          form.responseMode === FormResponseMode.Encrypt
+          form.responseMode === FormResponseMode.Encrypt &&
+          !form.get('newEncryptionBoundary')
```

Comment:
nit: `newEncryptionBoundary` is repeated several times in different files. Probably a good idea to create a constant for it so it's easier to track usages when we need to remove this flag

Reply:
Good point, thanks! Done in [6c0c3ab](https://github.com/opengovsg/FormSG/pull/6595/commits/6c0c3ab4917d6d9501cb5a23bd3c98bf853d2183)